Add Textbook Macros and Alternate Version

Pelonza opened the pull request:
I added a bunch of textbook macros, like you set up for Ben's comments, and a macro for my own valpo comments. Those files should merge without too many hick-ups. 

The other files included are an alternate version which splits the chapters up by "days" for apprxoimate coverage/assignment. I left them as a v2 instead of merging in since they are significantly different from the original textbook.

I sent a pull request to Ben also, but his fork sees to be way out of sync with yours.

Pelonza committed:
split out textbook and institutional comments

Modified all chapters to have more consistent and helpful
"side-comments". Specifically, added if-toggles in master for 3
textbooks (Thomas, Stewart, Larson), and a new comment: Valpo. Left any
"class/school" specific comments in 'bmw' but moved textbook problems to
appropriate textbook. Valpo comments should be all specific to valpo.

A few other (I hope) minor modifications to make it a little more
generally applicable to any school (some tweaks on pre-chapter intros,
and a few problem typos.)

Pelonza committed:
Version-2 splits chapters by day

The Version-2 Files have major changes:
Splits individual chapters into "days" with specific learning
objectives.
Removes or comments out several questions throughout chapters (to reduce
overall number)
Moves Limits to Chapter 13 (probably an unnecessary change, done more
for streamlining problem assignment)
Regrouped Chap 3+4 to break out material covered in other courses at
Valpo (Calc III only covers tail end of chap 4)

jasongrout commented:
Thanks!  I'm currently working in industry, so I'm very unlikely to teach multivariable calc in the near future.  It looks like you forked from last changes I made, so I'm happy to consider your fork the "real" repo.

Pelonza committed:
Fixed CalcIII_modified_master to Compile

The first version of edits now compiles properly
(CalcIII_modified_master). I had to revert Chapter 8, Line-Integrals to
Ben's original version, so that probably needs revised to actually match
the other chapters in this version.

I also added in a few EPS files that were missing and causing problems.

Pelonza committed:
Fixed Compile Issue with Day-Split Version

I fixed all the incorrect file-names in the 'includes' for the
"with-day-splits" versions (normal/instructor both). Everything should
compile correctly. Output PDFs are also now included.

There are a few additional comments included in the master files
describing what's going on.

Pelonza committed:
Major Reorganization of File Structures

The biggest changes in this commit is a large reorganization of moving
Ben Woodruff's original version into its own sub-folder and Karl
Schmitt's 2014 version into it's own folder.

Also, any other Tex or PDF files not clearly identifiable were moved to
a general history folder.

This commit also adds in a copy of the review guides for each chapter
used by Karl Schmitt.

Pelonza committed:
Cleanup and Revision of Main File

This adds/creates a new version of the compile file which actually
builds the textbook.

This file was based originally on the "splits" version, that is, version
2. It includes significantly more commenting and explanation of the
Latex Code-chunks that are in the main file.

Additionally there are some tweaks made to the wrap-up, copyright info
and other minor things in the pre-review pages.

Add's the "short" command version to all of the margin comment options
to allow some cleaner pages.

Pelonza committed:
Revised Review Chapter

Edited review chapter. Major change is removing it from Valpo-version to
be moved to vector section.

Also some more minor updates to the main file.

Pelonza committed:
Added Reference Doc

Started a reference text file that has all the problems and a short
description for all the problems which have explicit labels. This is to
help check confirm labels for cross-referencing.

Pelonza committed:
In-Process Vectors Chapter

Beginning of Mods to vector's chapter

Pelonza committed:
Revert "In-Process Vectors Chapter"

This reverts commit d5983c73c04788035aaba5a1fec2e9576b010e91.

Pelonza committed:
In-Progress Vector-Chap

In the stages of modifying Vector Chapter.

Pelonza committed:
Revised Vector Chapter

Finished Revising Vector Chapter. Some changes to other chapters as
well.

Pelonza committed:
Partial Changes to Chap4

Pelonza committed:
Unit 4: Polar Coordinates Revised

The unit "new coordinates" is edited and slightly reduced. This may end
up as not unit 4.

Pelonza committed:
Revised Chapter 5 - Functions

Updated and adjusted problems and some structure/ordering in chapter 5.
Also added several problems to the list of labeled problems. The list is
probably still missing a few from chapter 5.

Pelonza committed:
Updated Several Chapters

This is syncing the updates I did while travelling.

Pelonza committed:
Partial Edits of Derivative Chapter

unfinished edits completed in derivative chapter. I have to actually
think more....

Pelonza committed:
Finished Updating Derivatives

Pelonza committed:
General Fixing of things

Actually fixed the compiling of Units 0-5

Pelonza committed:
Merge pull request #1 from Pelonza/ShelterBranch

Shelterbranch
